### PR TITLE
Changed up the video feed set up in the base VC

### PR DESCRIPTION
**Pull request by jaimejiyepark**
* Moved the video setup to viewdidload to allow back and forth VC transition
* Moved ROI ui set up to viewDidAppear

### CODE REVIEW

File: CardScan/Classes/ScanBaseViewController.swift [hunk 1]
```diff
@@ -148,14 +148,15 @@ import Vision
...
         self.maskPreviewView(viewToMask: blurView, maskRect: frame)
-        
-        let borderWidth = CGFloat(5.0)
-        let cornersView = CornerView(frame: frame, borderWidth: borderWidth)
-        cornersView.layer.cornerRadius = self.regionCornerRadius + borderWidth
-        cornersView.layer.masksToBounds = true
-
-        cornersView.drawCorners(cornerColor: UIColor.green)
-        self.previewView?.addSubview(cornersView)
+//        print("during: \(frame)")
```
Comment (kingst) [inline CardScan/Classes/ScanBaseViewController.swift]: Let's remove this commented out code. It's in github so we can find it later if needed, no need to keep something around if we aren't going to use it.

File: CardScan/Classes/ScanBaseViewController.swift [hunk 2]
```diff
@@ -176,6 +177,9 @@ import Vision
         regionOfInterestLabel.layer.cornerRadius = self.regionCornerRadius
         regionOfInterestLabel.layer.borderColor = UIColor.white.cgColor
         regionOfInterestLabel.layer.borderWidth = 2.0
+
+        self.videoFeed.setup(captureDelegate: self, completion: { success in })
```
Comment (kingst) [inline CardScan/Classes/ScanBaseViewController.swift]: I'd like to see a comment about why this is here with a pointer to either apple docs or apple example code that does the same thing.

File: CardScan/Classes/ScanBaseViewController.swift [hunk 3]
```diff
@@ -197,17 +201,23 @@ import Vision
...
-                self.setupMask()
-            }
+        if self.ocr.numbers.count > 0 && self.ocr.expiries.count > 0 {
+            self.ocr.numbers.removeAll()
+            self.ocr.expiries.removeAll()
+            self.ocr.firstResult = nil
         }
-        
+        self.calledOnScannedCard = false
+        //MARK: ^Jaime just try the remove all
```
Comment (kingst) [inline CardScan/Classes/ScanBaseViewController.swift]: I don't understand what this comment means

File: CardScan/Classes/ScanBaseViewController.swift [hunk 4]
```diff
@@ -197,17 +201,23 @@ import Vision
...
         self.videoFeed.willAppear()
         self.isNavigationBarHidden = self.navigationController?.isNavigationBarHidden ?? true
         self.navigationController?.setNavigationBarHidden(true, animated: animated)
     }
     
+    open override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(true)
+        self.setupMask()
```
Comment (kingst) [inline CardScan/Classes/ScanBaseViewController.swift]: I'd love to see a comments + pointer to apple docs about why this is here.

Comment (jaimejiyepark) [reply CardScan/Classes/ScanBaseViewController.swift]: setUp mask is not a related to AVFoundation, it is our UI logic. What should i make a pointer to?

**Comment by kingst** [comment]
#shipit

**Pull request merged**